calibration/CTDMOG/CGINS-CTDMOG-13625__20170326.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13625,CC_ptcb1,-4.000000e-004,
 37-13625,CC_ptcb2,0.00E+00,
 37-13625,CC_ptempa0,1.673748e+002,
-37-13625,CC_ptempa1,-6.699173e-002,
+37-13625,CC_ptempa1,-6.699174e-02,
 37-13625,CC_ptempa2,-4.011635e-007,
-37-13625,CC_wbotc,-3.2028e-008,
+37-13625,CC_wbotc,-3.202790e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT